src/components/CommentList/CommentList.jsx

```diff
@@ -1,3 +1,12 @@
-export const CommentList = () => (
-  <>Put the list here</>
+import { CommentInfo } from '../CommentInfo';
+import './CommentList.scss';
+
+export const CommentList = ({ comments }) => (
+  <>
```

Comment:
You can remove the fragment '<>' and '</>' wrapping the 'div' since it is redundant in this context. This is because there's only one child inside and that's the 'div'.